cloud_functions.tf

```diff
@@ -12,6 +11,32 @@ locals {
   # common function for multiple actions
   cloud_internal_function_name = "${var.prefix}-${var.cluster_name}-weka-functions"
   function_ingress_settings    = var.subnet_autocreate_as_private ? "ALLOW_INTERNAL_ONLY" : "ALLOW_ALL"
+  vpc_connector_id             = var.vpc_connector_id == "" ? google_vpc_access_connector.connector[0].id : var.vpc_connector_id
+  project_number               = data.google_project.project.number
```

Comment:
lets call it `deployment_project_number` , basically I would like also the variable name to be `deployment_project_id`, but lets leave it for now.

Reply:
changed